app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -409,8 +418,8 @@
     <string name="download_archive_started">開始下載壓縮檔</string>
     <string name="download_archive_failure">下載失敗</string>
     <string name="archive_free">免費</string>
-    <string name="archive_original">原始</string>
-    <string name="archive_resample">重新採樣</string>
+    <string name="archive_original">原始檔案</string>
+    <string name="archive_resample">重新取樣檔案</string>
```

Comment:
These should be as short as possible because the button has limited width, see #1085